packages/codemods/src/command.ts

```diff
@@ -1,18 +1,41 @@
-export class CommandParserResult {
-  // path: string = ""; TODO this doesn't work with jest; why?
-  constructor(public shouldExit = false, public path = '') {}
+import yarr from 'yargs';
+import { Maybe, Nothing } from './maybe';
+export interface CommandParserResult {
+  shouldExit?: boolean;
```

Comment:
Recommend making this a `Maybe<ExitReason>`. ExitReason could be `type = 'success' | 'help' | 'badCommand' | 'programFailure'`

Reply:
Per my other comment I'll address this when I make an either. Not because I'll use either but because I think this should be batched there.